Firefox Reality issues get a browser-fixme label

Looks like the Firefox Reality issues get a `browser-fixme` label as well as a `browser-firefox-reality` one.
We sould not apply the `browser-fixme` label in these situations.

Here's a recent example:
https://github.com/webcompat/web-bugs/issues/22349

![image](https://user-images.githubusercontent.com/16573771/49656248-8cfedb00-fa45-11e8-9ea3-be80e7707475.png)

```diff
diff --git a/tests/unit/test_webhook.py b/tests/unit/test_webhook.py
index 2ff0202..c4aede9 100644
--- a/tests/unit/test_webhook.py
+++ b/tests/unit/test_webhook.py
@@ -164,6 +164,7 @@ class TestWebhook(unittest.TestCase):
              ({'browser': 'Firefox 30.0',
              'extra_labels': 'browser-focus-geckoview'}, 'browser-firefox'),
             ({}, 'browser-fixme'),
+            ({'browser': 'Mozilla/5.0 (Android 7.1.2; Mobile VR; rv:66.0) Gecko/66.0 Firefox/66.0'}, 'browser-firefox-reality'),  # noqa
         ]
         for metadata_dict, expected in metadata_tests:
             actual = helpers.extract_browser_label(metadata_dict)
```

that gives

```
 nosetests tests/unit/test_webhook.py
F..............
======================================================================
FAIL: Extract browser label name.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/karl/code/webcompat.com/tests/unit/test_webhook.py", line 171, in test_extract_browser_label
    self.assertEqual(expected, actual)
AssertionError: 'browser-firefox-reality' != 'browser-fixme'

----------------------------------------------------------------------
Ran 15 tests in 0.092s

FAILED (failures=1)
```

I guess we need to check that if a proper extra-label exists, we don't need to validate or extract what should be the label.

--- a/tests/unit/test_webhook.py
+++ b/tests/unit/test_webhook.py
@@ -70,6 +70,19 @@ class TestWebhook(unittest.TestCase):
         <!-- @browser: Firefox Mobile (Tablet) 40.0 -->
         """
 
+        self.issue_body5 = """
+        <!-- @browser: Android 8.1.0 -->
+        <!-- @ua_header: Mozilla/5.0 (Android 8.1.0; Mobile VR; rv:65.0) Gecko/65.0 Firefox/65.0 -->
+        <!-- @reported_with: browser-fxr -->
+        <!-- @extra_labels: browser-firefox-reality, type-media -->
+
+        **URL**: https://vrporn.com/closing-shift-shaft/
+
+        **Browser / Version**: Android 8.1.0
+        **Operating System**: Android 8.1.0
+        **Tested Another Browser**: Yes
+        """  # noqa
+
     def tearDown(self):
         """Tear down tests."""
         pass
@@ -198,7 +211,8 @@ class TestWebhook(unittest.TestCase):
         labels_tests = [
             (self.issue_body, ['browser-firefox', 'type-media', 'type-stylo']),
             (self.issue_body2, ['browser-fixme', 'type-foobar']),
-            (self.issue_body3, ['browser-firefox-tablet'])
+            (self.issue_body3, ['browser-firefox-tablet']),
+            (self.issue_body5, ['browser-firefox-reality', 'type-media']),
         ]
         for issue_body, expected in labels_tests:
             actual = helpers.get_issue_labels(issue_body)

--- a/webcompat/webhooks/helpers.py
+++ b/webcompat/webhooks/helpers.py
@@ -154,14 +154,17 @@ def get_issue_info(payload):
 
 def get_issue_labels(issue_body):
     """Extract the list of labels from an issue body to be sent to GitHub."""
-    metadata_dict = extract_metadata(issue_body)
-    browser_label = extract_browser_label(metadata_dict)
-    extra_labels = extract_extra_labels(metadata_dict)
-    priority_label = extract_priority_label(issue_body)
     labelslist = []
-    labelslist.extend([browser_label, priority_label])
+    metadata_dict = extract_metadata(issue_body)
+    extra_labels = extract_extra_labels(metadata_dict)
+    browser_label = extract_browser_label(metadata_dict)
     if extra_labels:
+        # if extra_labels contains a browser tag, we do not need to extract it
+        if any(label.startswith('browser') for label in extra_labels):
+            browser_label = None
         labelslist.extend(extra_labels)
+    priority_label = extract_priority_label(issue_body)
+    labelslist.extend([browser_label, priority_label])
     labelslist = [label for label in labelslist if label is not None]
     return labelslist
 

